docs/contributing/onboarding-dev.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Onboarding Dev Docs
```

Comment:
It's documentation, so let's use full words here: `Developer Onboarding` maybe?

Reply:
Sure